CONTRIBUTION.md

```diff
@@ -52,6 +62,9 @@ In general apply common sense to the Code you write. "Effective Java" is also al
 * Use Guava's `Preconditions` class for Argument validation
 * Validate `@NonNull` values with `Objects.requireNonNull` if necessary
 * use getters and setters, even in the declaring class
+* Avoid getters and setters for protected fields in non API-Code. 
```

Comment:
```suggestion
* Use protected fields in non-API superclasses, rather than private ones with getters and setters, if
      1. Those field are barely accessed in that superclass and are heavily accessed in its subclasses.
      2. There is no additional need for getters and setters, such as:
          1.  External access to the field .
          2. The need to run some additional code or logic when getting or setting the value of the field.
```
The point I was making was much more narrow that that. The crux of my argument was that without some additional need for getters/setters, if a field is hardly used the superclass and heavily used in its subclasses:
1.  Is effectively the property of the subclass, thus it makes sense not to have the superclass horde it away under complete privatized protection.
2. To privatize it means either:
    1. Declaring local variables in subclass methods, which just needlessly takes up space and requires needless overhead.
    2. Peppering the subclass methods with getters that accomplish nothing but add the word `get` and the symbols `()` everywhere. `getX()` is longer than `x`, and objectively just means adding unnecessary length to the lines of code that access the field.

The **_sole_** accomplishment of privatizing a protected field, in the case I described above, is to:
1. Add 2 methods (the getter & setter)
2. Add `get` and `()` everywhere.

The bottom line is that doing so simply adds unnecessary text without logical justification.